tests/phpunit/integration/class-test-case.php

```diff
@@ -173,4 +173,20 @@ protected function get_reflective_property( $property, $class_name ) {
 
 		return $property;
 	}
+
+	/**
+	 * Get the value of the private or protected property.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @param string $property Property name for which to gain access.
+	 * @param string $class_name Name of the target class.
+	 *
+	 * @return mixed
+	 * @throws \ReflectionException Throws an exception if property does not exist.
+	 */
+	protected function get_reflective_property_value( $property, $class_name ) {
```

Comment:
This method is less effective in the base Test Case.  Why?  Some class constructs require arguments to be passed into them for instantiating an object.  

Therefore, this method needs to be part of the individual test suite.  Let's remove it from here.